docs/guide/release-notes.asciidoc

```diff
@@ -0,0 +1,261 @@
+[[release-notes]]
+== Release notes
+
+* <<rn-7110>>
+* <<rn-7101>>
+* <<rn-7100>>
+* <<rn-791>>
+* <<rn-790>>
+* <<rn-781>>
+* <<rn-780>>
+* <<rn-771>>
+* <<rn-770>>
+* <<rn-760>>
+* <<rn-751>>
+* <<rn-750>>
+* <<rn-710>>
+* <<rn-705>>
+* <<rn-704>>
+* <<rn-703>>
+* <<rn-702>>
+* <<rn-701>>
+* <<rn-700>>
+
+
+[discrete]
+[[rn-7110]]
+=== 7.11.0 (2021-02-10)
```

Comment:
This is missing the `7.12.0` release notes, but so is master so I'll fix that up now.

Reply:
Added via b7e33a8